### ISSUE TITLE
Can’t support chinese enter.

### ISSUE THREAD

**Issue by wolf71** [rationalmatter/Juno-Issue-Tracker#27]
Can using keyboard or hardware keyboard enter chinese char. But can copy/paste some chinese chars, and display and run ok.
etc: print(“测试”)

 - Device: [iPad Pro 11"]
 - iOS version: [iOS 12.3.1]
 - App version [1.0.3]

**Comment by mieqq** [comment]
系统 bug，iOS 13 已修复

**Comment by navoshta** [comment]
Yep, this appears to be fixed in iOS 13. I'll see if I can come up with a workaround for iOS 12, otherwise just give it a couple of weeks until iOS 13 comes out.
